server.js

```diff
@@ -2,6 +2,23 @@ const express = require("express");
 const app = express();
 const cors = require("cors");
 const PORT = 4000;
+const sqlite = require("sqlite3");
+
+let db = new sqlite.Database("../shopping-list-database/database.sqlite", err => {
+    if (err) {
+        console.log(err.message)
+    } else {
+        console.log("Database connected");
+    }
+});
+//Already runned, table exist
+// db.run("CREATE TABLE grocery_items (name, isCompleted, id)");
```

Comment:
Would be great to run something like this every time the service starts. This would make sure the database have all the table(s) needed. Even if this is the first time the database is used (i.e. it is blank).

This should be done in the callback above (at the else branch where "database connected" is printed to make sure that the database is already open when this is run).

You can run something like this:

```sql
CREATE TABLE IF NOT EXISTS grocery_items ( ... );
```

This either create the table, or does nothing if the table already exist.